docs/Features/cypress.md

```diff
@@ -14,7 +14,9 @@ When generating your application, make sure to check "Cypress" in your optional
 
 ## How to use it
 
-Once your application is generated, you can optionally add some entities, it will auto-generate the corresponding tests for your entities. Execute cypress using this command :
+Once your application is generated, you can optionally add some entities, it will auto-generate the corresponding tests for your entities. Make sure to run it at least once then execute cypress using this command :
 ```bash
-src/YourApplication/ClientApp/node_modules/.bin/cypress open
-```
\ No newline at end of file
+cd src/YourApplication/ClientApp
+./node_modules/.bin/cypress open
```

Comment:
Isn't `npx cypress open` more simple?

Reply:
`npx cypress` only works with npm@5.2.0+ according to [cypress website](https://docs.cypress.io/guides/guides/command-line#How-to-run-commands).
In order to keep compatibility with earlier versions I decided to only put the .`/node_modules` ones. 

I will put both, with a link to this cypress wiki for reference.

Reply:
Yes, but can't we consider that this version is mandatory ?